code/datums/components/storage/concrete/pockets.dm

```diff
@@ -28,30 +28,18 @@
 	attack_hand_interact = TRUE // so the detectives would discover pockets in their hats
 
 /datum/component/storage/concrete/pockets/shoes
+	max_items = 2
 	attack_hand_interact = FALSE

```

Comment:
Why 2 items

Reply:
2 shoes

Reply:
two sides of the boot doggie

Reply:
Two items is also the default for shoes.

Reply:
As in, right now, we can hold two items in our shoes anyways.